frontend/web/styles/3rdParty/_hljs.scss

```
@@ -161,7 +162,7 @@
   font-family: $font-family-header;
   overflow: hidden;
   @include transition(all 0.2s ease-in-out);
-  color: $btn-code-button-color;
+  color: var(--color-text-default);
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)